common-test/pom.xml

```diff
@@ -50,6 +50,11 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+            <version>RELEASE</version>
```

Comment:
What is the scope of the dependency? Is it really `compile`? Or is it `test`?

Reply:
@scholzj  it is `test` https://github.com/json-path/JsonPath

Reply:
Could you make it explicit then in the pom.xml document?